packages/api-explorer/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@mia-platform/api-explorer",
   "description": "UI components for the API explorer",
-  "version": "6.0.1",
+  "version": "6.0.2-alpha.3",
   "main": "dist/index.js",
   "dependencies": {
     "@json-editor/json-editor": "^1.4.0-beta.0",
-    "@mia-platform/markdown": "^6.0.1",
-    "@mia-platform/oas-extensions": "^6.0.1",
-    "@mia-platform/syntax-highlighter": "^6.0.1",
-    "@mia-platform/variable": "^6.0.1",
+    "@mia-platform/markdown": "^6.0.2-alpha.1",
+    "@mia-platform/oas-extensions": "^6.0.2-alpha.1",
+    "@mia-platform/syntax-highlighter": "^6.0.2-alpha.1",
+    "@mia-platform/variable": "^6.0.2-alpha.1",
+    "choices.js": "^9.0.1",
```

Comment:
Why choices.js and not a react select? e.g. antd select?

Reply:
this select is rendered by json-editor, which is not a react library. In order to use a react-component as a component of json-editor I think that should be mounted the component with React.createElement but would be hard to manage the management of this react-component with the json-editor life-cycle and the react one